test/e2e/apps/rc.go

```diff
@@ -390,6 +391,33 @@ var _ = SIGDescribe("ReplicationController", func() {
 			return err
 		})
 	})
+
+	ginkgo.It("should get and update a ReplicationController scale", func() {
+		rcClient := f.ClientSet.CoreV1().ReplicationControllers(ns)
+		rcName := "e2e-rc-" + utilrand.String(5)
+
+		ginkgo.By(fmt.Sprintf("Creating ReplicationController %q", rcName))
+		rc := newRC(rcName, int32(1), map[string]string{"name": rcName}, WebserverImageName, WebserverImage, nil)
```

Comment:
is 1 the replica Count here ? Should we make explicit by using a variable called `initialRCReplicaCount` or something similar ?

Reply:
Yes, changed.